commands/events.py

```diff
@@ -78,7 +78,7 @@ def template(self, event, submission:str=None):
             timediv = datetime.utcnow() - datetime.utcfromtimestamp(report["report_time"])
             hours, remainder = divmod(int(timediv.total_seconds()), 3600)
             minutes, seconds = divmod(remainder, 60)
-            error = "<@{}> Looks like {} beat you to the punch. Better luck next time {}".format("SNIPED!" if minutes < 2 else "<:dupebutton:341981924010491904>", self.participants[str(report["author_id"])], event.author.id)
+            error = "{} Looks like {} beat you to the punch. Better luck next time <@{}>".format("SNIPED!" if minutes < 2 else "<:dupebutton:341981924010491904>", self.participants[str(report["author_id"])], event.author.id)
```

Comment:
Maybe instead you could change it to:
```python
if minutes < 2 else event.author.id, self.participants[str(report["author_id"])], "<:dupebutton:341981924010491904>")
```

That would instead put the emoji at the end (as I assume it's meant to be)?

Reply:
the original fix he made makes it how i intended it to be, sniped or dupe at the front and a mention at the back, makes more sense to me that way, i just put the <@> in the wrong place

Reply:
Hm, interesting @AEnterprise. Maybe it'd be worth moving the mentions to the end of all messages, then, for the sake of consistency? 🙂

Reply:
Maybe both could be in the front to keep the consistency?

Reply:
it could be in front but then things just need some rewording, i thought this would be better in this case at the end, i can fix that with the other bugs i need to squash otherwise

Reply:
I'm happy to do a PR to fix the rest to save you some time, if you want?

Reply:
no worries, i can fix them all myself in one go together with some of the other bugs from the event

Reply:
Sounds good! 🙂